extensions/wikidata/module/styles/dialogs/schema-template-dialog.less

```diff
@@ -0,0 +1,15 @@
+.schema-template-list {
+     list-style: none;
```

Comment:
For .schema-template-list, add a `max-height:450px` and `overflow-y:auto` in case list gets too long.